java/mtsj/batch/pom.xml

```diff
@@ -14,6 +14,8 @@
 
   <properties>
     <java.version>1.8</java.version>
+
+    <sonar.coverage.jacoco.xmlReportPaths>${basedir}/../${aggregate.report.dir}</sonar.coverage.jacoco.xmlReportPaths>
```

Comment:
why do we need this redundant for every module? Can we do that in the parent POM?
Why do we have to configure this at all? Did something change in SonarQube so it is not detected by default? Or is the problem that we ourselves are violating conventions and configuring the report in the wrong directory?

Reply:
IMHO related to the aggregation you are using. Is that required or recommended?

Reply:
I used this guide: https://community.sonarsource.com/t/coverage-test-data-importing-jacoco-coverage-report-in-xml-format/12151

> However, the JaCoCo plugin imports coverage reports module by module, so we need to import the same report multiple times (once for every module) to have coverage for all modules. To achieve this we set the property sonar.coverage.jacoco.xmlReportPaths in every module